src/dex/aave-v3/config.ts

```diff
@@ -65,12 +64,12 @@ export const Config: DexConfigMap<DexParam> = {
       poolAddress: AaveV3BNB.POOL,
       wethGatewayAddress: AaveV3BNB.WETH_GATEWAY,
     },
-    [Network.ZKEVM]: {
-      ethGasCost: 246 * 100,
-      lendingGasCost: 328 * 1000,
-      poolAddress: AaveV3PolygonZkEvm.POOL,
-      wethGatewayAddress: AaveV3PolygonZkEvm.WETH_GATEWAY,
-    },
+    // [Network.ZKEVM]: {
```

Comment:
Why do we remove zk?

Reply:
@JackieJoo please check comment from @sakulstra in the original PR https://github.com/paraswap/paraswap-dex-lib/pull/856#discussion_r1881157417